ports back the cit disabling of regen jelly no long permanently changing someone's hair color

Elli-Skala opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

regen jelly no longer makes your hair permanently purple 

## Why It's Good For The Game

on tg its just a slight thing that no one really cares about, but on a player base like ours being someone with white fur but you're given what's support to be a healing chem just permanently changes your hair to be a mismatched purple. changing the color of some one is such a minor thing but its the major reason its avoided on this server. we already removed it back on old base years ago because we saw the problem then. this wasn't suppose to be such a major detriment, but in our environment it is. 

## Changelog
:cl:
del: removed regen jelly permanently making your hair purple 
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ZephyrTFA commented on code/modules/reagents/chemistry/reagents/medicine_reagents.dm:
```diff
@@ -1067,13 +1067,11 @@
 		return
 
-	var/mob/living/carbon/human/exposed_human = exposed_mob
+	/* var/mob/living/carbon/human/exposed_human = exposed_mob
```
```suggestion
	/** SKYRAT EDIT BEGIN - Regen Jelly no longer overrides hair color
	var/mob/living/carbon/human/exposed_human = exposed_mob
```

ZephyrTFA commented on code/modules/reagents/chemistry/reagents/medicine_reagents.dm:
```diff
@@ -1067,13 +1067,11 @@
-	// SKYRAT EDIT ADDITION BEGIN
-	exposed_human.update_mutant_bodyparts(force_update=TRUE)
-	// SKYRAT EDIT END
+	exposed_human.update_hair() */ skyrat edit 
```
```suggestion
	exposed_human.update_hair()
**/ // SKYRAT EDIT END 
```

ZephyrTFA commented:
Doesn't follow the contributing guidelines correctly.
Please see code suggestions

GoldenAlpharex commented:
Hard no from me. Regen jelly is quite good and having your hair color changed is what "balances" it so people don't overuse it.

Also, whilst it's technically permanent, you can still fix it _very_ easily with Genetics.

This is power creep and just not a really good change, as it removes the only drawback regenerative jelly has.

I also fixed mis-matching parts in a previous PR, for people that only have one hair color (non-anthros) so, if anything, I'd consider this one of the very few chem that has an unique side-effect that makes for some interesting twists in roleplay.

RatFromTheJungle commented:
if you think it's an actual issue, genetics can just reset your DNA to fix it, but the PR removing R-J's only drawback, (and not even nerfing the healing) is a tad much, this makes it just objectively better omnizine

Elli-Skala commented:
> Hard no from me. Regen jelly is quite good and having your hair color changed is what "balances" it so people don't overuse it.
> 
> Also, whilst it's technically permanent, you can still fix it _very_ easily with Genetics.
> 
> This is power creep and just not a really good change, as it removes the only drawback regenerative jelly has.
> 
> I also fixed mis-matching parts in a previous PR, for people that only have one hair color (non-anthros) so, if anything, I'd consider this one of the very few chem that has an unique side-effect that makes for some interesting twists in roleplay.
>if you think it's an actual issue, genetics can just reset your DNA to fix it, but the PR removing R-J's only drawback, (and not even nerfing the healing) is a tad much, this makes it just objectively better omnizine
I think I agree with a lot of this. it is interesting and i kind of like it, its just doesn't fit well in the furry player base (cit knew this and changed it years ago) we have. though while yes a genetists could fix it we've limited the slot to one, gutted the role to the point a lot of the time we don't even have that one slot, and a large chuck of the geneists we do have seem to not even know the dna trick (let alone people who get the chem and wants the hair color changed back) it tends to last whole rounds. personaly id prefer if i could do a way to code it is make it last for the duration of the chem in your system but im not sure if hair coloring even suports that

Elli-Skala commented:
> Hard no from me. Regen jelly is quite good and having your hair color changed is what "balances" it so people don't overuse it.
> 
> Also, whilst it's technically permanent, you can still fix it _very_ easily with Genetics.
> 
> This is power creep and just not a really good change, as it removes the only drawback regenerative jelly has.
> 
> I also fixed mis-matching parts in a previous PR, for people that only have one hair color (non-anthros) so, if anything, I'd consider this one of the very few chem that has an unique side-effect that makes for some interesting twists in roleplay.

thought the balance idea was that the "difficulty" in getting it (ie xenobio caring enough to give you the stuff/ stuff to make it) 
on the main tg base i doubt half as many people even care about their hair color then they do here

GoldenAlpharex commented:
> > Hard no from me. Regen jelly is quite good and having your hair color changed is what "balances" it so people don't overuse it.
> > Also, whilst it's technically permanent, you can still fix it _very_ easily with Genetics.
> > This is power creep and just not a really good change, as it removes the only drawback regenerative jelly has.
> > I also fixed mis-matching parts in a previous PR, for people that only have one hair color (non-anthros) so, if anything, I'd consider this one of the very few chem that has an unique side-effect that makes for some interesting twists in roleplay.
> 
> I think I agree with a lot of this. it is interesting and i kind of like it, its just doesn't fit well in the furry player base (cit knew this and changed it years ago) we have. though while yes a genetists could fix it we've limited the slot to one, gutted the role to the point a lot of the time we don't even have that one slot, and a large chuck of the geneists we do have seem to not even know the dna trick (let alone people who get the chem and wants the hair color changed back) it tends to last whole rounds. personaly id prefer if i could do a way to code it is make it last for the duration of the chem in your system but im not sure if hair coloring even suports that
>
> thought the balance idea was that the "difficulty" in getting it (ie xenobio caring enough to give you the stuff/ stuff to make it)
> on the main tg base i doubt half as many people even care about their hair color then they do here

It's still its only downside and is very, _very_ easy to produce, and if I recall correctly, has no OD, unlike Omnizine.

I actually have a completely opposite opinion, I actually believe that this downside is perfect for a roleplay-oriented codebase, as it doesn't really have any mechanic consequence, it's all just appearance.

It literally takes 5 seconds to do in Genetics, and if there's no Geneticist, just get the RD to let you in. It's a really interesting roleplay opportunity and this would just make Genetics even more irrelevant.

Elli-Skala commented:
> I actually have a completely opposite opinion, I actually believe that this downside is perfect for a roleplay-oriented codebase, as it doesn't really have any mechanic consequence, it's all just appearance.

on a roleplay server yeah honestly it would be cool
just not a furry one. its a weird chem where i have to warn other players about its effects due to being used to the old cit effects and its down side not being listed on the wiki. because unlike any other healing chem, this is one you have to get consent for to use even if they were in crital, if not you have to face having to deal with another ooc quantum hair dye issue

FlamingLily commented:
> Hard no from me. Regen jelly is quite good and having your hair color changed is what "balances" it so people don't overuse it.

I hate that this is a reasonable and correct statement for this server. But yes, this is true/

Gandalf2k15 commented:
Feature freeze. No balance changes.
